apps/cbportal/base/CBGrid/.macros/page/versionselect/3_version_select.py

```diff
@@ -5,7 +5,7 @@
 def main(j, args, params, tags, tasklet):
     scl = j.clients.osis.getNamespace('system')
     params.result = page = args.page
-    url = scl.grid.search({"id":j.application.whoAmI.gid}).get('manifestUrl')
+    url = scl.grid.searchOne({"id":j.application.whoAmI.gid}).get('manifestUrl')
```

Comment:
manifestUrl should be under settings

Reply:
so no need to add it to the models ?

Reply:
https://github.com/jumpscale7/jumpscale_core7/pull/73